app.config.ts

```diff
@@ -25,6 +25,12 @@ const config: ExpoConfig = {
   web: {
     favicon: './assets/favicon.png',
   },
+  owner: 'papers',
```

Comment:
Cratera, no?

Reply:
Not yet, I have to rename it but I will address this on my next PR where I'll be checking out [this](https://docs.expo.dev/develop/development-builds/introduction/)